release-notes.md

```diff
@@ -183,18 +194,108 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v22.0
+
+This release has been tracked through the [v21.0
+project](https://github.com/cloud-hypervisor/cloud-hypervisor/projects/24).
+
+### GDB Debug Stub Support
+
+Cloud Hypervisor can now be used as debug target with GDB. This is controlled
+by the `gdb` compile time feature and details of how to use it can be found in
+the [gdb
+documentation](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/docs/gdb.md).
+
+
+### `virtio-iommu` Backed Segments
+
+In order to facilitate hotplug devices that require being behind an IOMMU (e.g.
+QAT) there is a new option `--platform iommu_segments=<list_of_segments>` that
+will place all the specified segments behind the IOMMU.
+
+### Before Boot Configuration Changes
+
+It is now possible to change the VM configuration (e.g. add or remove devices,
+resize) before the VM is booted.
+
+
+### `virtio-balloon` Free Page Reporting
+
+If `--balloon free_page_reporting=on` is used then the guest can report pages
+that is it not using to the VMM. The VMM will then notify the host OS that
+those pages are no longer in use and can be freed. This can result in improved
+memory density.
+
+### Support for Direct Kernel Booting with TDX
+
+Through the use of `TD-Shim` lightweight firmware it is now possible to
+directly boot into the kernel with TDX. The [TDX
+documentation](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/docs/intel_tdx.md#tdshim)
+has been updated for this usage.
+
+### PMU Support for AArch64
+
+A PMU is now available on AArch64 for guest performance profiling. This will be
+exposed automatically if available from the host.
+
+### Documentation Under CC-BY-4.0 Licese
```

Comment:
```suggestion
### Documentation Under CC-BY-4.0 License
```

Reply:
I did manage to spot & fix it in the version that went in the release notes.

Reply:
Cool!